dist/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <title>To do list</title>
+<script defer src="main.js"></script></head>
+<body>
+  <section class="to-do-list">
+    <div class="title">
+      <h2>Today's to do</h2>
+    </div>
+    <div class="write-task">
+      <input type="text" class="input-text" placeholder="Add your list ..." required>
+      <i class="far fa-plus-square"></i>
+    </div>
+    <div class="the-list"></div>
+      <button class="clean-completed">Clear All Completed</button>
+    </section>
+</body>
+
+</html>
```

Comment:
following the HTML good practice an extra free line is always required at the end of the code